requirements.txt

```diff
@@ -1,3 +1,3 @@
 prefect>=2.8.2
 distributed==2022.2.0; python_version < '3.8'
-distributed>=2022.5.0,<=2023.3.1; python_version >= '3.8' # upper bound due to issue with get_client starting in 2023.3.2 - https://github.com/dask/distributed/issues/7763
+distributed>=2022.5.0,!=2023.3.2,!=2023.3.2.1,!=2023.4.*,!=2023.5.*; python_version >= '3.8' # don't allow versions from 2023.3.2 to 2023.5 (inclusive) due to issue with get_client starting in 2023.3.2 (fixed in 2023.6.0) - https://github.com/dask/distributed/issues/7763
```

Comment:
I don't believe this excludes 2023.3.3 through < 2023.4.*

Reply:
I think it should be like `>=2022.5.0,<=2023.3.1,>=2023.6.0`

Reply:
`2023.3.2.1` was the last `distributed` release in March

Reply:
👍